### DISCUSSION TITLE
Lifepath is broken at upbringing

### DISCUSSION THREAD

**Discussion by AltheusII** [bcholmes/StarTrek2d20#166] [General]
Hi, I think the character generator is broken at the upbringing screen, when I try to get beyond that stage I get told that I have to select a discipline, even though I have already chosen a discipline.

This is a recent change - this worked correctly last week.

**Comment by bcholmes** [comment]
You're right. That's a bug. I'll get a fix for you quickly.

**Reply by bcholmes** [reply]
Thanks for taking the time to report it.
